src/api/controllers/products.controller.ts

```diff
@@ -0,0 +1,92 @@
+import { APIRequestContext } from '@playwright/test';
+import { RequestApi } from 'api/apiClients/request';
+import { IProduct, IProductsResponse, IRequestOptions } from 'types';
+import { apiConfig } from 'config';
+import { IProductResponse } from 'types';
+import { convertRequestParams, logStep } from 'utils';
+
+export class ProductsController {
+  private request: RequestApi;
+  constructor(context: APIRequestContext) {
+    this.request = new RequestApi(context);
+  }
+  @logStep('Send getByID request')
+  async getById(id: string, token: string) {
```

Comment:
Отступов много не бывает, а глазу легче смотреть на логически разделенные блоки
```suggestion
  private request: RequestApi;
  
  constructor(context: APIRequestContext) {
    this.request = new RequestApi(context);
  }
  
  @logStep('Send getByID request')
  async getById(id: string, token: string) {
```

Reply:
fixed